.github/workflows/build_and_test.yml

```diff
@@ -1098,3 +1098,14 @@ jobs:
           cd ui-test
           npm install --save-dev
           node --experimental-vm-modules node_modules/.bin/jest
+
+  maven-test:
```

Comment:
fine to me, let's revert this change after test pass

Reply:
Okay.

Reply:
I will revert this change, because the maven test has passed and the verification is successful.

Reply:
Done.